projects/kubernetes/kubernetes/build/lib/binaries.sh

```diff
@@ -13,14 +13,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-function build::binaries::use_go_version_k8s() {
+function build::binaries::get_go_version_k8s() {
     local -r releasebranch="$1"
 
     if [[ "$releasebranch" == "1-18" ]]; then
-        build::common::use_go_version "1.13"
```

Comment:
Can we maybe put a GOLANG_VERSION file in the various release branch directories instead?

Reply:
I like that idea.  Kubernetes is the only project so far using different go versions per release branch and this was the pattern put in place for that.  I can open a follow up that moves it to a GOLANG_VERSION.